vendor/github.com/libvirt/libvirt-go/BUILD.bazel

```diff
@@ -81,11 +83,12 @@ go_library(
         "stream_wrapper.h",
         "typedparams.go",
     ],
+    cgo = True,
     cdeps = [
         "//:libvirt-libs",
         "//:libvirt-headers",
     ],
-    cgo = True,
+    copts = ["-Ibazel-out/k8-fastbuild/genfiles"],
```

Comment:
As discussed offline, let's make  sure that when people run `make deps-update`, that they can still build. [buildozer](https://github.com/bazelbuild/buildtools/tree/master/buildozer) can be used to inject the things we need.